taskcluster/arm64-build.sh

```diff
@@ -21,7 +21,7 @@ do_bazel_build
 
 do_deepspeech_binary_build
 
-export SUPPORTED_PYTHON_VERSIONS="3.7.6:ucs4"
+export SUPPORTED_PYTHON_VERSIONS="3.7.6:ucs2"
```

Comment:
My bad, it is related: the definition was wrong, in fact. Because of the mess around py stuff, we actually built proper `abiflags` but the testing environment was inconcsistent

Reply:
Still seems like it'd be better in a separate commit, with a message explaining what you just commented here?